### ISSUE TITLE
PCIe form factor definition is missing

### ISSUE THREAD

**Issue by ulfglaeser** [ni/nixnet-python#253]
In nixnet._enums the form factor is declared as:
```
class DevForm(enum.Enum):
    '''Device physical form factor.'''
    PXI = _cconsts.NX_DEV_FORM_PXI
    PCI = _cconsts.NX_DEV_FORM_PCI
    C_SERIES = _cconsts.NX_DEV_FORM_C_SERIES
    PX_IE = _cconsts.NX_DEV_FORM_PX_IE
    USB = _cconsts.NX_DEV_FORM_USB
```
However, the PCIE form factor is missing (introduced to the C API in NI-XNET 17.0):
`    PCIE = _cconsts.NX_DEV_FORM_PCIE`

**Comment by ulfglaeser** [comment]
Ahh, it looks like it's just missing from the documentation (your link), but it is in the nixnet.h file.

**Comment by epage** [comment]
Turns out we made the API from NI-XNET 16.1

See https://github.com/ni/nixnet-python/issues/255

**Comment by jashnani** [comment]
Fixed in #263
